### PR TITLE
Alpha docs for CLI

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adding alpha documentation for the Cloud CLI

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- `dbt run`
+- `dbt build`
+- `dbt deps`
+- `dbt cancel`
+- `dbt reattach`
+- `dbt --help`
+
+## Using dbt Cloud CLI
+
+**More informaiton coming soon**
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
**Coming soon**
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+brew install dbt-cloud-cli
+```
+
+Verify the installation:
+
+```bash
+which dbt
+dbt --help
+```
+
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md:30]: Could we make "upgrading" an aside or a callout? It currently feels like a necessary next step.

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+:::warning Alpha functionality 
+
+The following installation instructions are for the dbt Cloud CLI, currently in alpha. These instructions are not intended for general audiences at this time. 
+
+::: 
+
+## Installing dbt Cloud CLI
+
+### Install and update with Brew (recommended)
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md]: "### Install and update with Brew on MacOS (recommended)"

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+Upgrade the CLI:
+
+```bash
+brew update
+brew upgrade dbt-cloud-cli
+```
+
+### Manually install (Windows only)
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md]: "### Manually install (Windows and Linux)"

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+1. Download the latest darwin release from [Github](https://github.com/dbt-labs/dbti/releases).
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md]: "Download the latest release for your platform from [Github](https://github.com/dbt-labs/dbt-cli/releases)."

Note that the url changed

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+2. Place the `dbt` executable in `/usr/local/bin` or `/usr/.local/bin` if `/usr/local/bin` doesn’t exist.
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md]: Replace 2+3 with:
"Add the `dbt` executable to your path."

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+3. Navigate to `/usr/local/bin` / `/usr/.local/bin` in Finder and double click on the `dbt` executable:
+    1. You will receive a message that the executable is untrusted and cannot be run. Override this by right-clicking the executable and selecting “Open”. A new prompt will pop up, allowing you to open the executable.
+    2. After clicking “Open”, a terminal window will appear. The window can be closed, and you can now invoke `dbt` from any other terminal*.
+        - You may need to deal with conflicts due to other `dbt` installations. Run `which dbt` to see the location of the `dbt` installation you’re using. If you get `/usr/local/bin/dbt` from `which dbt`, you’re using the one you just installed. If not, you may need to uninstall other installations, directly invoke the executable, or handle path conflicts differently, such as using an alias for the new executable.
+4. Move to a directory with a dbt project, and create a `dbt_cloud.yml` file containing your `project-id` from dbt Cloud.
+5. Invoke `dbt --help` from your terminal to see a list of supported commands.
+
+#### Updating dbt Cloud installation (Windows only)
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md]: Change to "#### Updating dbt Cloud installation (Windows + Linux)"

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+Follow the same process in [Installing dbt Cloud CLI](#manually-install-windows-only) and replace the existing `dbt` executable with the new one. You should not have to go through the security steps again.
+
+## Setting up the CLI
+
+1. Ensure that you have created a project in dbt Cloud.
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md]: Add link to https://cloud.getdbt.com/

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+2. Ensure that your personal [development credentials](#glossary) are set on the project
+
+3. Create an environment variable with your [dbt cloud API key](#glossary):
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md]: Replace link to point to "https://cloud.getdbt.com/settings/profile#api-access"

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+
+#### Updating dbt Cloud installation (Windows only)
+
+Follow the same process in [Installing dbt Cloud CLI](#manually-install-windows-only) and replace the existing `dbt` executable with the new one. You should not have to go through the security steps again.
+
+## Setting up the CLI
+
+1. Ensure that you have created a project in dbt Cloud.
+
+2. Ensure that your personal [development credentials](#glossary) are set on the project
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md]: Add link to https://cloud.getdbt.com/settings/profile/credentials

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+5. Navigate to a dbt project
+
+```bash
+   > $ cd ~/dbt-projects/jaffle_shop
+```
+
+6. Ensure that a `dbt_cloud.yml` file exists in the project directory. The file is required to have a `project-id` field with a valid [project ID](#glossary):
+
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md]: "You can find your project id by selecting your project and clicking on "documentation" in the navigation bar.  Your project id is the number in the url: https://cloud.getdbt.com/documentation/1/projects/{project id}.

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+- `dbt reattach`
+- `dbt --help`
+
+## Using dbt Cloud CLI
+
+**Coming soon**
+
+### Glossary
+
+- **dbt cloud API key:** your API key found by navigating to the **gear icon**, clicking **Profile Settings**, and scrolling down to **API**.
```
Comment (greg-mckeon) [inline website/docs/docs/cloud/cloud-cli-installation.md:105]: Nice, didn't see these.  I think we can just link for the cloud API key, but keep the other ones in the glossary.

**Review by greg-mckeon — CHANGES_REQUESTED** [review]
took a pass! let me know if anything is unclear

File: website/docs/docs/cloud/cloud-cli-installation.md (hunk 3 above)
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
### Install and update with Brew on MacOS (recommended)
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+Upgrade the CLI:
+
+```bash
+brew update
+brew upgrade dbt-cloud-cli
+```
+
+### Manually install (Windows only)
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
### Manually install (Windows and Linux)
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+Upgrade the CLI:
+
+```bash
+brew update
+brew upgrade dbt-cloud-cli
+```
+
+### Manually install (Windows and Linux)
+
+1. Download the latest darwin release from [Github](https://github.com/dbt-labs/dbti/releases).
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
1. Download the latest darwin release from [Github](https://github.com/dbt-labs/dbt-cli/releases).
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 15]
```diff
@@ -0,0 +1,111 @@
...
+2. Place the `dbt` executable in `/usr/local/bin` or `/usr/.local/bin` if `/usr/local/bin` doesn’t exist.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
2. Add the `dbt` executable to your path.
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 16]
```diff
@@ -0,0 +1,111 @@
...
+3. Navigate to `/usr/local/bin` / `/usr/.local/bin` in Finder and double click on the `dbt` executable:
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
3. Navigate to your `dbt` path in Finder and double-click on the `dbt` executable:
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 17]
```diff
@@ -0,0 +1,111 @@
...
+    1. You will receive a message that the executable is untrusted and cannot be run. Override this by right-clicking the executable and selecting “Open”. A new prompt will pop up, allowing you to open the executable.
+    2. After clicking “Open”, a terminal window will appear. The window can be closed, and you can now invoke `dbt` from any other terminal*.
+        - You may need to deal with conflicts due to other `dbt` installations. Run `which dbt` to see the location of the `dbt` installation you’re using. If you get `/usr/local/bin/dbt` from `which dbt`, you’re using the one you just installed. If not, you may need to uninstall other installations, directly invoke the executable, or handle path conflicts differently, such as using an alias for the new executable.
+4. Move to a directory with a dbt project, and create a `dbt_cloud.yml` file containing your `project-id` from dbt Cloud.
+5. Invoke `dbt --help` from your terminal to see a list of supported commands.
+
+#### Updating dbt Cloud installation (Windows only)
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
#### Updating dbt Cloud installation (Windows + Linux)
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 18]
```diff
@@ -0,0 +1,111 @@
...
+
+Follow the same process in [Installing dbt Cloud CLI](#manually-install-windows-only) and replace the existing `dbt` executable with the new one. You should not have to go through the security steps again.
+
+## Setting up the CLI
+
+1. Ensure that you have created a project in dbt Cloud.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
1. Ensure that you have created a project in [dbt Cloud](https://cloud.getdbt.com/).
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 19]
```diff
@@ -0,0 +1,111 @@
...
+
+2. Ensure that your personal [development credentials](#glossary) are set on the project
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
2. Ensure that your personal [development credentials](https://cloud.getdbt.com/settings/profile/credentials) are set on the project.
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 20]
```diff
@@ -0,0 +1,111 @@
...
+
+3. Create an environment variable with your [dbt cloud API key](#glossary):
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
3. Create an environment variable with your [dbt cloud API key](https://cloud.getdbt.com/settings/profile#api-access):
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 21]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+5. Navigate to a dbt project
+
+```bash
+   > $ cd ~/dbt-projects/jaffle_shop
+```
+
+6. Ensure that a `dbt_cloud.yml` file exists in the project directory. The file is required to have a `project-id` field with a valid [project ID](#glossary):
+
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
You can find your project ID by selecting your project and clicking on **documentation** in the navigation bar. Your project ID is the number in the URL: https://cloud.getdbt.com/documentation/1/projects/{project id}.
```

File: website/docs/docs/cloud/cloud-cli-installation.md (hunk 19 above)
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md:57]:
```suggestion
2. Ensure that your personal [development credentials](https://cloud.getdbt.com/settings/profile/credentials) are set on the project
```

File: website/docs/docs/cloud/cloud-cli-installation.md (hunk 18 above)
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md:55]:
```suggestion
1. Ensure that you have created a project in [dbt Cloud](https://cloud.getdbt.com/).
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 22]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+Verify the installation:
+
+```bash
+which dbt
+dbt --help
+```
+
+Upgrade the CLI:
```
Comment (matthewshaver) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
### Upgrade the CLI
```

**Review by greg-mckeon — APPROVED** [review]

**Pull request merged**